scripts/buf_lint_check.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+jar_file_path=$?
+config_file_path=$?
+os_type=$?
+github_actions_path=$1
+
+check_protobuf() {
+  echo "********************************"
+  echo "Checking Protobuf file formatting"
+  echo "********************************"
+
+  $jar_file_path lint --config $config_file_path
+
+  status=$?
+
+  if [ "$status" = 0 ] ; then
+    echo "Protobuf lint check completed successfully"
+    exit 0
+  else
+    echo "********************************"
+    echo "Protobuf lint check issues found"
+    echo "Please fix the above issues."
+    echo "********************************"
+    exit 1
+  fi
+}
+
+windows_warning() {
+  echo "********************************"
+  echo "Currently Buf check doesn't support on Windows"
+  echo "********************************"
+}
+
+create_jar_file_path() {
```

Comment:
Is this actually creating anything, or is it setting global variables? Perhaps instead: populate_jar_config_file_paths would be clearer.

Reply:
Done